pkgs/servers/web-apps/phpmyadmin/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl }:
+stdenv.mkDerivation rec {
+  pname = "phpmyadmin";
+  version = "5.0.4";
+
+  src = fetchurl {
+    url = "https://files.phpmyadmin.net/phpMyAdmin/${version}/phpMyAdmin-${version}-all-languages.tar.gz";
+    sha256 = "+7mTt0p8Kc4vzba2viKwtsWL/43zKSq0L2KAhxRQwxY=";
+  };
+
+  phases = [
+    "unpackPhase"
+    "installPhase"
+  ];
+
+  installPhase =
+  ''
+    mkdir -p $out
+    cp -r * $out/
+  '';
+
+  meta = {
+    homepage = https://www.phpmyadmin.net/;
+    description = "phpMyAdmin is a free and open source administration tool for MySQL and MariaDB";
```

Comment:
`description` shouldn't start with the name of the package. See nixpkgs (not NixOS) manual for full details.